src/App.tsx

```diff
@@ -12,11 +12,19 @@ import {
 import { PageLoadingSpinner } from './components/Spinner';
 import { DivingCylinderSetManagement } from './views/DivingCylinderSetSettings';
 import UserManagement from './views/UserManagement';
-import { User } from './components/User/user';
+import { FC, useEffect, useState } from 'react';
+import getCookie from './components/common/GetCookie';
+import { getUser, User } from './lib/apiRequests/userRequests';
+import { AXIOS_CONFIG } from './lib/apiRequests/api';
+import { UserComp } from './components/User/user';
 
 const QUERY_CLIENT = new QueryClient();
 
-const Content = (): JSX.Element => {
+type ContentCompProps = {
+  user: User | undefined;
+};
+
+const Content: FC<ContentCompProps> = (props): JSX.Element => {
```

Comment:
Ei passata käyttäjää propsina muille sivuille ainakaan nyt näin alkuun. Tehdään käyttäjän hakeminen muihin näkymiin omana tikettinään todennäköisesti react queryn tilanhallintaa hyödyntäen